.github/workflows/ets-from-source.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         with:
           edm-version: ${{ env.INSTALL_EDM_VERSION }}
       - name: Install click to the default EDM environment
-        run: edm install -y wheel click coverage
+        run: edm install -y click coverage
```

Comment:
`coverage` should be removed here - there's no value having it in the bootstrap environment. When we run tests, we're using the `coverage` package from the target environment.